crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -1,11 +1,8 @@
+pub use crate::Withdrawal as StandaloneWithdraw;
```

Comment:
not a fan of this alias and the term `standalone` all the functions in general.

this should just be Withdrawal

Reply:
Ok , I'll use scoped use then